src/include/kdbprivate.h

```diff
@@ -518,10 +518,13 @@ int elektraValidateKeyName(const char *name, size_t size);
  */
 #define ELEKTRA_MAX_ARRAY_SIZE (21)
 
+#if !defined(__cplusplus)
```

Comment:
I did not talk about to int64_t -> kdb_long_long_t but about the ```#if !defined(__cplusplus)```. Why should the compiler confuse the types in C++?